README.md

```diff
@@ -41,4 +41,15 @@ The crawler will just update the data, starting with the oldest timestamp.
 
 ## Database Stucture
 
-![System Diagram](/assets/images/DBStructure.png)
\ No newline at end of file
+![System Diagram](/assets/images/DBStructure.png)
+
+## Update Institutions
+
+Once someone has updated the github_repos.json file and the pull request was merged, the new or updated insitution must be added to the database.
+
+This can be done in 4 easy steps:
+
+1. Pull Repository
+2. Connect to DB with MongoDB Compass
```

Comment:
Do we have setup instructions for new developers? Installing MongoDB Compass is easy, but which DB do they connect to, and where to find the auth credentials?